lottie/src/main/java/com/airbnb/lottie/animation/LPaint.java

```diff
@@ -35,4 +38,13 @@ public LPaint(int flags, PorterDuff.Mode porterDuffMode) {
   public void setTextLocales(@NonNull LocaleList locales) {
     // Do nothing.
   }
+
+  @Override public void setAlpha(int alpha) {
```

Comment:
Could you add a doc explaining why this is necessary so future people can understand?

Reply:
It's not a "memory issue". It's avoiding memory allocations.